<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWidget</class>
 <widget class="QWidget" name="RegisterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Đăng ký tài khoản</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        background-color: #f0f0f0;
        font-family: Arial;
    }
    QLabel {
        color: #333333;
    }
    QLineEdit {
        padding: 8px;
        border: 1px solid #999;
        border-radius: 4px;
        background-color: white;
        min-height: 20px;
    }
    QPushButton {
      padding: 8px 16px;
      border-radius: 4px;
      min-width: 100px;
      background-color: #2196F3;
      color: white;
      border: none;
    }
    QPushButton#registerButton {
        background-color: #34a853;
    }
    QPushButton#registerButton:hover {
        background-color: #2d8544;
    }
    QPushButton#registerButton:disabled {
        background-color: #cccccc;
    }
    QPushButton#backButton {
        background-color: #757575;
    }
    QPushButton#backButton:hover {
        background-color: #616161;
    }
    QRadioButton {
        color: #333333;
        spacing: 8px;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ĐĂNG KÝ TÀI KHOẢN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>15</number>
     </property>
     <property name="verticalSpacing">
      <number>15</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="usernameLabel">
       <property name="text">
        <string>Tên đăng ký:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="usernameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Mật khẩu:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="roleLabel">
       <property name="text">
        <string>Vai trò:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="roleLayout">
       <item>
        <widget class="QRadioButton" name="studentRadio">
         <property name="text">
          <string>Sinh viên</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="teacherRadio">
         <property name="text">
          <string>Giáo viên</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="firstNameLabel">
       <property name="text">
        <string>Họ:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="firstNameLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lastNameLabel">
       <property name="text">
        <string>Tên:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lastNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>Quay lại</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="registerButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Đăng ký</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>